### ISSUE TITLE
Background fails to be set if absolute path isn't specified.

### ISSUE THREAD

**Issue by gatoWololo** [RAR27/dyn-wall-rs#8]
Thank you for the awesome program! I plan to start using exclusively!

At least in Ubuntu when I try:
```
> dyn-wall-rs -a temp2/
temp2/boo.png has been set as your wallpaper
```
But my background turns completely dark. This is because `dyn-wall-rs` generates the call:
```
gsettings set org.gnome.desktop.background picture-uri 'file://temp2/boo.png'
```
Which will silently succeed as `gsettings` doesn't report this as an error, but `gsettings` only works when the absolute path is specified. The "fix" is for me to include the full path from `/`.

I think `dyn-wall-rs` should check if the path the user passed is an absolute path and if not, either report an error or you can use [std::fs::canonicalize](https://doc.rust-lang.org/std/fs/fn.canonicalize.html) to fetch the absolute path.

**Comment by rehanzo** [comment]
This is something that I noticed as well, and I'll get to fixing it. Thank you for bringing up canonicalize. I was not aware of it.

**Comment by rehanzo** [comment]
Fixed this on 7eda5711488db6699e57332cb14150aa0fdf5881. You can download the linux binary [here](https://github.com/RAR27/dyn-wall-rs/suites/783481022/artifacts/8291756), if you would like. Again, thank you for bringing up canonicalize, and thank you for using my little program!
